backend/src/database/migrations/R__organizationEnrichmentMaterializedViews.sql

```diff
@@ -0,0 +1,22 @@
+-- drop old materialized view if exists
+drop materialized view if exists "organizationsGlobalActivityCount" cascade;
+
+-- create the materialized view
+create materialized view "organizationsGlobalActivityCount" as
+select 
+    osa."organizationId",
+    sum(osa."activityCount") as total_count
```

Comment:
NIT: maybe call this total_count_estimate since summing all these together won't give the real count bcs af overlaps